test/blackbox-tests/test-cases/pkg/opam-package-cycle-with-or.t

```diff
@@ -0,0 +1,29 @@
+We test an repository with a package cycle that has an alternative solution which avoids a
+cycle. We have the following packages:
+
+>      a --> b --> c
+
+Now c depends either on a or on a fourth package d. Therefore there is a valid solution
+available that avoids a cycle.
+
+  $ . ./helpers.sh
+  $ mkrepo
+
+  $ mkpkg a <<EOF
+  > depends: [ b ]
```

Comment:
Shouldn't it be `[ "b" ]`, otherwise opam interprets it as a variable?

Reply:
You're totally right. The actual behaviour seems to be to just ignore the cycle.